lmnet/lmnet/utils/predict_output/output.py

```diff
@@ -35,12 +34,13 @@ class JsonOutput():
     Please see [Output Data Specification](https://github.com/LeapMind/lmnet/wiki/Output-Data-Specification).
     """
 
-    def __init__(self, task, classes, image_size, data_format):
+    def __init__(self, task, classes, image_size, data_format, bench={}):
```

Comment:
Don't use mutable object as an argument's default value
http://go/python-review-guidelines#using-listdict-as-the-default-arg

Reply:
Thanks, it is reasonable. Fixed.